packages/board-format/src/f-puzzles.ts

```diff
@@ -88,7 +88,7 @@ type FPuzzlesQuadruple = {
 
 
 
-const fpuzzlesSizes = {
+export const fpuzzlesSizes = {
```

Comment:
Could we move this constant to `board-utils` and rename it to something like `gridToBoxSizeMap`? https://github.com/SudokuStudio/SudokuStudio/blob/dev/packages/board-utils/src/board-utils.ts
(Unless you can think of a better place for it?)